packages/site/src/providers/AtlantisPreviewEditorProvider.tsx

```diff
@@ -360,23 +360,25 @@ const skeletonHTML = (theme: Theme, type: "web" | "mobile") => {
 <head>
 <style>
 html,body,#root {
-  height: 100%;
-  width: 99%;
+  height: 100vh;
+  width: 100vw;
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  min-height: 200px;
+  min-height: 280px;
+  margin: 0;
+  padding: var(--space-smallest);
```

Comment:
Minus this small space needed to prevent the very edge of things like cards from being clipped